client/src/pages/Map/Map.js

```diff
@@ -99,6 +99,8 @@ export default function Map({
         style: 'mapbox://styles/waterthetrees/ckyckqkqz8e4b14rn3rm1hh9k',
         center: [-122.34725, 37.7343787],
         zoom: 10,
+        maxZoom: 18.5,
```

Comment:
I found that when the map zoom was exactly the max zoom on the layers, the layers would disappear, so I maxed out the map zoom to .01 less than the layer max zoom.